docs/admin/external-auth.md

```diff
@@ -94,6 +94,19 @@ CODER_EXTERNAL_AUTH_0_CLIENT_SECRET=xxx
 CODER_EXTERNAL_AUTH_0_AUTH_URL=https://bitbucket.domain.com/rest/oauth2/latest/authorize
 ```
 
+### Bitbucket Cloud
+
+Bitbucket Cloud requires the following environment variables:
+
+```env
+CODER_EXTERNAL_AUTH_0_TYPE="bitbucket-cloud"
+CODER_EXTERNAL_AUTH_0_ID=bitbucket
+CODER_EXTERNAL_AUTH_0_CLIENT_ID=xxx
+CODER_EXTERNAL_AUTH_0_CLIENT_SECRET=xxx
+CODER_EXTERNAL_AUTH_0_AUTH_URL=https://bitbucket.domain.com/site/oauth2/authorize
+CODER_EXTERNAL_AUTH_0_TOKEN_URL=https://bitbucket.domain.com/site/oauth2/access_token
```

Comment:
Can bitbucket cloud be hosted on any other domain but `https://bitbucket.org/`?

https://github.com/coder/coder/blob/ced6ae01b77613f0145e356158cb1a36e1334104/coderd/externalauth/externalauth.go#L621-L622

I know bitbucket-server has custom domain and self hosting support.

Reply:
From what I could find, there seems to be an [issue](https://jira.atlassian.com/browse/BCLOUD-11257) requesting this feature which remains open since 2015. Couldn't find it in the [roadmap](https://www.atlassian.com/roadmap/cloud?selectedProduct=bitbucket&search=domain) either

Reply:
That is different. That allows web hosting of a repository that has some static `index.html`.

The values being configured here are for OIDC, which is a different feature. Even if you were to host web pages using the CNAME, the OIDC values would still be the `bitbucket.org` domain.